<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertPixImageClass</class>
 <widget class="QMainWindow" name="ConvertPixImageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConvertPixImage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>951</width>
      <height>571</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QTableWidget" name="tableWidgetPixFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>931</width>
       <height>481</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FileName</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButtonAddPixFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>+ Add file</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddPixFolder">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>+ Add files in folder</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>951</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>PIX file</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDeleteList">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>37</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>- Delete selected lists</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>951</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>951</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Image path</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditOutputPath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>881</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFindPath">
     <property name="geometry">
      <rect>
       <x>902</x>
       <y>30</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonExecuteConvert">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>670</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute convert</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxExtension">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>680</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PNG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BMP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JPG</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>670</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
